tests/integration_tests.rs

```diff
@@ -0,0 +1,45 @@
+use std::process::Command;
+use wait_timeout::ChildExt;
+
+use std::fmt;
+use std::env;
+use std::fs::remove_dir_all;
+use std::path::PathBuf;
+use std::time::Duration;
+use std::{thread, time};
+
+
+#[test]
+fn test_nginx() {
```

Comment:
I think for the integrations simpler might be easier to do it directly with a shell script that can run in CircleCI.
(rather than a rust test itself, that we need to compile first to execute later).

Thoughts?

Reply:
I went ahead and switched it to a shell script.